src/extraction/wikidata.py

```diff
@@ -200,23 +201,28 @@ def read_entities(
                             id_to_attrs[unique_id]["aliases"] = []
                             aliases = obj["aliases"]
                             if aliases:
-                                lang_aliases = aliases.get(lang, None)
-                                if lang_aliases:
-                                    for item in lang_aliases:
-                                        id_to_attrs[unique_id]["aliases"].append(
-                                            item["value"]
-                                        )
+                                # Merge aliases in this language with aliases in English. Most aliases are specified in
+                                # English, much less so in any other languages. Often these other languages still refer
+                                # to an entity with an English alias or one that is similar to it, so it makes sense to
+                                # still consider English aliases.
```

Comment:
This is true for typical named entities like names of people, cities, etc. But it wouldn't be good to do this sort of thing if people are using Wikidata for more general concepts like https://www.wikidata.org/wiki/Q14748 - so I'd suggest putting this functionality behind a feature flag.

Reply:
Opt-in or opt-out for this feature?

Reply:
Erm. Default set to `True`, maybe? It feels like that might be the more frequent setting?

Reply:
Added in https://github.com/explosion/wikid/pull/16/commits/6866fc14c3e406de7f19db308f4a6b7cc1655e87.